src/packages/client/src/generation/utils.ts

```diff
@@ -133,7 +135,7 @@ export function getDefaultArgName(
 export function getOperation(action: DMMF.ModelAction): 'query' | 'mutation' {
   if (
     action === DMMF.ModelAction.findMany ||
-    action === DMMF.ModelAction.findUnique || 'findOne'
+    action === DMMF.ModelAction.findUnique
```

Comment:
I guess that's a fix unrelated to linting

Reply:
It's a bit weird that "'findOne'" was only removed here?

Reply:
The `|| 'findOne'` is an always truthy statement, so it was invalid altogether. Result from linting